### DISCUSSION TITLE
Welcome to mu-haskell Discussions!

### DISCUSSION THREAD

**Discussion by serras** [higherkindness/mu-haskell#274] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about, share ideas and suggestions for new and current features.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by kutyel** [comment]
We are happy to help if anyone has doubts regarding the usage of mu-haskell! 👋🏻
